google/cloud/pubsub/internal/tracing_message_batch_test.cc

```diff
@@ -127,6 +128,7 @@ TEST(TracingMessageBatch, Flush) {
   auto mock = std::make_unique<pubsub_testing::MockMessageBatch>();
   EXPECT_CALL(*mock, Flush).WillOnce([] {
     EXPECT_TRUE(ThereIsAnActiveSpan());
+    EXPECT_TRUE(OTelContextCaptured());
     return [](auto) {};
```

Comment:
I think you can test that the context is detached by: `EXPECT_FALSE(OTelContextCaptured())` in this lambda.

Reply:
Done